testsuite/forge/src/interface/public.rs

```diff
@@ -65,24 +87,35 @@ pub enum Coffer<'t> {
 }
 
 pub trait Fund {
-    fn fund(&mut self, currency: Currency, auth_key: AuthenticationKey, amount: u64) -> Result<()>;
+    fn fund(
+        &mut self,
+        currency: Currency,
+        auth_key: AuthenticationKey,
+        amount: u64,
+        tx_factory: TransactionFactory,
+    ) -> Result<()>;
```

Comment:
Mind if we just revert these changes to the Fund trait for now. It doesn't really seem necessary atm